### PR TITLE
Stacks and queues

### PR DESCRIPTION
**Pull request by yxd3549**
A funny comment

### CODE REVIEW

**Comment by eml5244** [comment]
nerd

**Review by eml5244 — CHANGES_REQUESTED** [review]
NERD

**Review by pmm4516 — APPROVED** [review]
wicked

**Review by eml5244 — APPROVED** [review]

File: CS1/Stacks-And-Queues/drivethru_sol.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        return False
+
+
+def print_drivethru(drivethru):
+    """
+    Prints the current state of the driveway
+    Because of the nature of stacks, 
+    we can only print the top car and the size of the driveway
+    """
+    if is_empty(drivethru):
```
Comment (yxd3549) [inline CS1/Stacks-And-Queues/drivethru_sol.py:35]: Like this

**Pull request merged**